cmd/podman/umount.go

```diff
@@ -36,6 +46,7 @@ func umountCmd(c *cli.Context) error {
 	}
 	defer runtime.Shutdown(false)
 
+	force := c.Bool("force")
```

Comment:
This is probably a stupid question. In some places, when a 1-letter shortcut is defined for an option, I see:
```
    x := c.Bool("longname") || c.Bool("x")
```
(Seems to be more prevalent with `c.IsSet` than `Bool`). I just want to make sure that the above line also recognizes `-f`. No need to respond if it does.